lib/fun_with_flags/store/serializer/redis.ex

```diff
@@ -5,7 +5,7 @@ defmodule FunWithFlags.Store.Serializer.Redis do
 
   @type redis_hash_pair :: [String.t]
 
-  @spec serialize(FunWithFlags::Gate.t) :: redis_hash_pair
+  @spec serialize(fun_with_flags :: Gate.t()) :: redis_hash_pair
```

Comment:
Ah, good catch!

Just one thing. This is actually me accidentally using Ruby's module navigation operator. I wrote `FunWithFlags::Gate` but I actually meant `FunWithFlags.Gate`. So this just fixes the compiler warning, but actually breaks the typespec.

Can you please just change it to `FunWithFlags.Gate.t`?